kvmagent/kvmagent/kvmagent.py

```
@@ -10,17 +10,26 @@
 import pprint
-import functools
-import sys
-import commands
 import platform
+import re
+import subprocess
+import sys
+import traceback
 
+# TODO: -> python3: delete it
 reload(sys)
```

Ruff (0.15.5)

[error] Undefined name `reload`

(F821)